infrastructure/annotations-api/src/main.ts

```diff
@@ -123,43 +120,6 @@ class AnnotationsAPI extends TerraformStack {
     });
   }
 
-  /**
-   * Creates the elasticache and returns the node address list
-   * @param scope
-   * @private
-   */
-  private static createElasticache(
```

Comment:
@kschelonka would love a verification, but i can't see this used anywhere and in aws it has no cache values currently.

Reply:
I think it was just a template/copy-paste/intended to be used at some point